apps/docs/src/app/core/component-docs/table/examples/table-custom-columns-example/table-custom-columns-example.component.html

```diff
@@ -0,0 +1,28 @@
+<table fd-table [keys]="displayedColumns">
```

Comment:
when hovering over one of the filter options and not the filter, it should use grab icon as cursor

Reply:
It's only example